src/EbmlElement.cpp

```diff
@@ -427,10 +427,9 @@ EbmlElement *EbmlElement::CreateElementUsingContext(const EbmlId & aID, const Eb
   // elements at the current level
   for (unsigned int ContextIndex = 0; ContextIndex < EBML_CTX_SIZE(Context); ContextIndex++) {
     if (aID == EBML_CTX_IDX_ID(Context,ContextIndex)) {
-      auto ClassInfos = EBML_CTX_IDX(Context,ContextIndex);
-      if (AsInfiniteSize && !ClassInfos.GetCallbacks().CanHaveInfiniteSize())
+      if (AsInfiniteSize && !EBML_CTX_IDX_INFO(Context,ContextIndex).CanHaveInfiniteSize())
```

Comment:
When merging this PR to the current master, this line doesn't compile anymore:

```
lib/libebml/src/EbmlElement.cpp:430:70: fatal error: no member named 'CanHaveInfiniteSize' in 'libebml::EbmlSemantic'
      if (AsInfiniteSize && !EBML_CTX_IDX_INFO(Context,ContextIndex).CanHaveInfiniteSize()) 
```

Reply:
Rebased on top of #267 which has a bit more macro usage and fixes that issue.